test/byok_test.go

```diff
@@ -40,3 +40,8 @@ func (s *CLITestSuite) TestBYOKDescribe() {
 		s.runIntegrationTest(tt)
 	}
 }
+
+func (s *CLITestSuite) TestBYOKAutocomplete() {
```

Comment:
```suggestion
func (s *CLITestSuite) TestByokAutocomplete() {
```

Reply:
I made this one uppercase since the other test functions here do this (same for api key; only one function isn't uppercase). Should I switch them all to lowercase (first letter capital), or save that for a later PR?